src/coreComponents/constitutive/unitTests/testDruckerPrager.cpp

```diff
@@ -225,13 +225,13 @@ void testDruckerPragerExtendedDriver()
   StrainData data;
   data.strainIncrement[0] = -1e-3;
   real64 invariantP, invariantQ;
-  real64 total = 0;
+//  real64 total = 0;
```

Comment:
```suggestion
```
@joshua-white Was there a plan for this variable?

Reply:
You can delete this.  It was part of some debugging `printf` statements at line 231 during development, but you can delete all this now.